react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,46 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
-      : null
+  const { id } = orderForm
 
-  const getHeader = (mobile: boolean) => {
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    async function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      
+      const data = await fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions).then(res => res.json())
+      
+      if (!!data?.clientProfileData?.customerClass) {
+        setCustomerClass(data.clientProfileData.customerClass)
+      }
+      
+    }
+
+    getCustomerClass()
+  }, [])
+
+  const getClientName = (user: any) => {
```

Comment:
Also, user type isn't any, is an object that can have `name` property as a string or can be null.

Reply:
@igorbrasileiro Changed the type of the user from any to Client.